### PR TITLE
UX-534 Add footer

### PR DESCRIPTION
**Pull request by jonambas**

### CODE REVIEW

File: lib/api.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { groq } from 'next-sanity';
 import { getClient } from '@lib/sanity';
 
-export const PAGE_TYPES = `['page', 'foundation', 'component', 'content', 'pattern', 'brand', 'resource', 'update']`;
+export const PAGE_TYPES = `['page', 'foundation', 'component', 'content', 'pattern', 'brand', 'resource', 'update', 'menus']`;
```
Comment (logansparlin) [inline lib/api.ts:4]: I don't think we need 'menus' here unless you had a specific reason for adding it

File: studio/structure/index.js [hunk 2]
```diff
@@ -23,22 +22,23 @@ const DRAFTS_QUERY = `* [_type == 'workflow.metadata' && state == 'draft'] {
   )
 }`;
 
+const FooterSettings = S.listItem()
```
Comment (logansparlin) [inline studio/structure/index.js:25]: Good idea

**Review by logansparlin — APPROVED** [review]
Looks great! Just one small comment

File: lib/api.ts (hunk 1 above)
Comment (jonambas) [reply lib/api.ts:4]: Oh no, i don't even know what this does

**Pull request merged**